Expensify/App
# [Hold] fixed spaces collapsing in Messages

### Details

https://github.com/Expensify/Expensify.cash/issues/3859#issuecomment-872606868

### Fixed Issues

$ Fixes #3859

### Tests
1. Do `npm install` to get the latest packages used.
2. Open E.cash local server.
3. Go to any conversation and send the following message.
```
0 space
 1 spaces
  2 spaces
   3 spaces
    4 spaces
```

### QA Steps

 1. Log in to e.cash and navigate to a conversation
 2. Send the following text
```
0 space
 1 spaces
  2 spaces
   3 spaces
    4 spaces
```
3. Output should contain the same amount of spacing as added.

### Tested On


### Screenshots


#### Web | Desktop

![image](https://user-images.githubusercontent.com/24370807/124371016-49bb8600-dc9b-11eb-9396-eb53c80a4b98.png)

#### Mobile Web

![Screen Shot 2021-07-04 at 07 59 52](https://user-images.githubusercontent.com/24370807/124371278-d8c99d80-dc9d-11eb-9a31-61909cb0fee8.png)

#### iOS


#### Android

![Screenshot 2021-07-04 07:58:25](https://user-images.githubusercontent.com/24370807/124371259-a91a9580-dc9d-11eb-9411-b3b022276ecc.png)
